lib/carin_for_blue_button_test_kit/client/v2.0.0/collection.rb

```diff
@@ -7,12 +7,13 @@ module CarinForBlueButtonTestKit
                          Practitioner: ['c4bb-Practitioner'],
                          RelatedPerson: ['c4bb-RelatedPerson'] }.freeze
 
-  SEARCHES_BY_PRIORITY = { Coverage: [['_id'], ['_lastUpdated']],
+  SEARCHES_BY_PRIORITY = { Coverage: [['_id'], ['_lastUpdated'], ['patient']],
```

Comment:
Where are you coming up with this list of required searches? It doesn't seem in-sync with what's in the [Carin BB capability statement](https://hl7.org/fhir/us/carin-bb/STU2/CapabilityStatement-c4bb.html#resource--details), which even though it is for the server is the best guidance I've seen on what search parameters are required. Is there somewhere else you're looking?

Reply:
I am also looking at the US Core 3.1.1 required searches for these resources